### DISCUSSION TITLE
Reader mode CSS clues?

### DISCUSSION THREAD

**Discussion by vossenjp** [lwindolf/liferea#1281] [Q&A]
I'm running Liferea 1.13.7 on Linux Mint 21.2 and I like "reader mode" but I hate the default gray background because of the low contrast to black text.  I've previously hacked `~/.config/liferea/liferea.css` with:
```
...
/* Item/feed description */
// div.content { }
div.content {
   background-color: white;
}
...
```
But that has no effect on "reader mode."  Does anyone have any CSS clues for fixing this?

**Comment by vossenjp** [comment]
Wow, I took a wild guess from comments in https://unix.stackexchange.com/a/128600 and tried this, and it worked.  Is there a better or more targeted way?
```
...
body {
    background: white;
}
...
```

**Reply by lwindolf** [reply]
Actually setting `div.content` does work for me, both with and without Reader mode. I do not know why it did not in your case.

For easier debugging launch Liferea with CLI param `--debug-html` so you can use the Webkit inspector (right click in HTML view and choose "Inspect", also ensure to move the inspector to a separate window so all input works).
